src/libstd/fs.rs

```diff
@@ -844,6 +844,9 @@ pub fn symlink_metadata<P: AsRef<Path>>(path: P) -> io::Result<Metadata> {
 
 /// Rename a file or directory to a new name.
 ///
+/// Moving a file or directory is also a rename, in a sense, and so this
+/// function is not limited a single directory.
```

Comment:
Er, to be clear, the issue states:

> std::fs::rename is missing the explicit description that you can also move a file outside of the current folder with it, as long as it's on the same file system.

This documentation doesn't indicate anything related to filesystem boundaries though? (there's also still the mentioning "rename" twice issue I'm worried about)